code/datums/helper_datums/pconstruction_datum.dm

```diff
@@ -0,0 +1,260 @@
+#define FORWARD -1
+#define BACKWARD 1
+
+/*
+This only exists so I don't break mecha construction
+because the system mecha uses sucks horribly
+
+so i'm using paradise's superior construction datum
+
+which looks 100x better in practice and is just generally better
+*/
+
+/datum/pconstruction
```

Comment:
Also, having just looked at and confirm this, the pconstruction system is literally just an older version of ```/datum/construction``` ported from either an old version of us or from /vg/

we're not going to accept an outdated version of something we already have.
I'd like you to convert pods to our existing ```/datum/construction``` system.